server/model/feed.go

```diff
@@ -19,24 +19,24 @@ package model
 //
 // swagger:model feed
 type Feed struct {
-	Owner    string `json:"owner"                   xorm:"repo_owner"`
-	Name     string `json:"name"                    xorm:"repo_name"`
-	FullName string `json:"full_name"               xorm:"repo_full_name"`
+	Owner    string `json:"owner"                   xorm:"feed_repo_owner"`
+	Name     string `json:"name"                    xorm:"feed_repo_name"`
+	FullName string `json:"full_name"               xorm:"feed_repo_full_name"`
 
-	Number   int64  `json:"number,omitempty"        xorm:"pipeline_number"`
-	Event    string `json:"event,omitempty"         xorm:"pipeline_event"`
-	Status   string `json:"status,omitempty"        xorm:"pipeline_status"`
-	Created  int64  `json:"created_at,omitempty"    xorm:"pipeline_created"`
-	Started  int64  `json:"started_at,omitempty"    xorm:"pipeline_started"`
-	Finished int64  `json:"finished_at,omitempty"   xorm:"pipeline_finished"`
-	Commit   string `json:"commit,omitempty"        xorm:"pipeline_commit"`
-	Branch   string `json:"branch,omitempty"        xorm:"pipeline_branch"`
-	Ref      string `json:"ref,omitempty"           xorm:"pipeline_ref"`
-	Refspec  string `json:"refspec,omitempty"       xorm:"pipeline_refspec"`
-	Remote   string `json:"remote,omitempty"        xorm:"pipeline_remote"`
-	Title    string `json:"title,omitempty"         xorm:"pipeline_title"`
-	Message  string `json:"message,omitempty"       xorm:"pipeline_message"`
-	Author   string `json:"author,omitempty"        xorm:"pipeline_author"`
-	Avatar   string `json:"author_avatar,omitempty" xorm:"pipeline_avatar"`
-	Email    string `json:"author_email,omitempty"  xorm:"pipeline_email"`
+	Number   int64  `json:"number,omitempty"        xorm:"feed_pipeline_number"`
+	Event    string `json:"event,omitempty"         xorm:"feed_pipeline_event"`
+	Status   string `json:"status,omitempty"        xorm:"feed_pipeline_status"`
+	Created  int64  `json:"created_at,omitempty"    xorm:"feed_pipeline_created"`
+	Started  int64  `json:"started_at,omitempty"    xorm:"feed_pipeline_started"`
+	Finished int64  `json:"finished_at,omitempty"   xorm:"feed_pipeline_finished"`
+	Commit   string `json:"commit,omitempty"        xorm:"feed_pipeline_commit"`
+	Branch   string `json:"branch,omitempty"        xorm:"feed_pipeline_branch"`
+	Ref      string `json:"ref,omitempty"           xorm:"feed_pipeline_ref"`
+	Refspec  string `json:"refspec,omitempty"       xorm:"feed_pipeline_refspec"`
+	Remote   string `json:"remote,omitempty"        xorm:"feed_pipeline_remote"`
+	Title    string `json:"title,omitempty"         xorm:"feed_pipeline_title"`
+	Message  string `json:"message,omitempty"       xorm:"feed_pipeline_message"`
+	Author   string `json:"author,omitempty"        xorm:"feed_pipeline_author"`
+	Avatar   string `json:"author_avatar,omitempty" xorm:"feed_pipeline_avatar"`
+	Email    string `json:"author_email,omitempty"  xorm:"feed_pipeline_email"`
```

Comment:
What a crazy model. I guess we should clean it up at some point as well.

Reply:
@anbraten well it act's like a `view`

Reply:
and I prefixed it, so if we refactor stuff it wont break

Reply:
Sure. I was mainly talking about all the different properties as I guess most of them are not even part of our UI atm.

Reply:
well in an dedicated refactor we can remove all not needed once and expose those who do make sense to the ui